docs/Software Architecture Document/Component Interaction View/assets/app_launch.plantuml

```diff
@@ -0,0 +1,49 @@
+@startuml
+/'
+It is a source file for SDL SAD sequence diagram.
+For further information, please, follow http://plantuml.com/ 
+'/
+
+title "Business layer - App Launch"
+
+participant "Connection Handler" as CH #DDD
+participant "Application Manager" as AM #DDD
+participant "App Launch" as AL
+participant "Resumption" as Resum
+
+== Device connected ==
+
+CH -> AM : Notification
+AM -> AL : Device Connected
+AL -> AL : Get apps data for device
+loop 
+	AL -> Resum : Get HMI level for app
+	AL <-- Resum 
+end
+loop 
```

Comment:
@LevchenkoS,
Maybe better to merge it in one loop?
And add comment like 'for each saved app'.